flash_flow.yml.erb.example

```diff
@@ -85,7 +85,24 @@ branch_info_file: 'your/random/file'
 
 #release:
 #  class:
-#    name: 'FlashFlow::Release::Percy'
+#    name: 'FlashFlow::Release::PercyClient'
 #    token: <%= ENV['PERCY_TOKEN'] %>
 #    repo: # Your github repo. For example, the flash_flow repo is 'flashfunders/flash_flow'
 #    api_url: 'https://percy.io/api/v1/'
+#    email: 'foo@bar.com'
+#    cc_email: 'bar@foo.com'
+#smtp:
+#  settings:
+#    domain: 'flashdough.com'
+#    address: 'smtp.sendgrid.net'
```

Comment:
please could we make this an environment variable?
The hostname may be about to change, if we move to Hubspot as our email provider.